webflow-hacks/misc-hacks/13/13.ts

```diff
@@ -0,0 +1,25 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const BUTTON_JS_SELECTOR = '[fs-hacks-element="hack13-back-1"]';
+  const BUTTON_JQUERY_SELECTOR = '[fs-hacks-element="hack13-back-2"]';
```

Comment:
I don't understand why are the selectors called JS and JQUERY

Reply:
sorry bad autocomplete. Fixing....